### ISSUE TITLE
Light font issue

### ISSUE THREAD

**Issue by TomaszGasior** [brendt/stitcher.io#135]
Please don't use light font weight for the content of your web page. It's really hard to read it on Linux. Just use font-weight: 400 and everything will be readable for everyone on every OS. :)

https://work.lisk.in/2020/07/18/font-weight-300.html

![imgg](https://user-images.githubusercontent.com/1395027/114862167-7e334a80-9dee-11eb-8aa1-c2f94e336d7b.png)

**Comment by brendt** [comment]
Thanks, I fixed it! Does this have to do with the pixel density of your screen?

**Comment by TomaszGasior** [comment]
It's related to pixel density (I have 1x only currently) but also to the fact that Mac OS makes fonts much bolder than Windows/Linux.

**Comment by brendt** [comment]
Is it better now?

**Comment by brendt** [comment]
(you might need to do a hard refresh due to browser caching)

**Comment by TomaszGasior** [comment]
Much better. Thank you. :)

![Zrzut ekranu z 2021-04-16 11-58-36](https://user-images.githubusercontent.com/1395027/115008196-20186d00-9eab-11eb-827b-eda4f54e053e.png)

**Comment by brendt** [comment]
Which font is used in the body btw? I don't recognise it 😂

**Comment by TomaszGasior** [comment]
This font is Cantarell. Your stylesheet `font-family: Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"` does not use it at all but I have aliased `Helvetica`, `sans-serif` and `-apple-system` to `Cantarell` in my local configuration of fontconfig. 

I checked how your site looks in Windows 10 and clean Fedora 33 installation: they both use font which just does not have light variant so the issue was hidden, still existing. 

-----

The reason why I have this kind of overrides in my configuration is because there is a trend in current web design to use system UI font, which is the font your operating system is using for displaying UI in native OS apps. There is no way to tell in CSS "use system UI" so website authors hardcode some font names like `-apple-system`, `Segoe UI`, `Roboto`. Sometimes they even care about Ubuntu. Most of the time they ignore default font in non-Ubuntu Linux distros which is Cantarell in OS with GNOME like Fedora and Oxygen (I my knowledge is up to date…) in OS with KDE desktop, like openSUSE.

There was a attempt to create standard way to say "use system UI font" in form of `font-family: system-ui` but it failed spectacularly because of some issues on Chinese installations of Microsoft Windows. Big sites like Medium  and GitHub used `font-family: system-ui` for a while but rolled back to hard-coded list because of that issue. 

Fedora configuration aliases fake `system-ui` font to `Cantarell` but because of that MS Windows issue no one wants to use `system-ui` for now. I aliased `-apple-system` to `Cantarell` because that's the fake font which Apple enforces people to use to get MacOS font: it cannot be imported directly by name. Because of that I can alias it to the system UI font of my OS and get the same level of experience like users of other OS-es for which there is hard-coded font name in most implemenation of the system UI CSS. 

I also aliased `Helvetica` to `Cantarell` because some implementations  of hard-coded system UI list fall back just to `Helvetica` (just before `sans-serif`) without specifying `Cantarell`. Cantarell has various variants of weights, including light. So, because of my local configuration the issue was easy to see.
